doc/includes/cli-reference.md

```diff
@@ -130,7 +130,7 @@ and untrack all branches.
 ### gs repo sync
 
 ```
-gs repo (r) sync (s)
+gs repo (r) sync (s) [flags]
```

Comment:
The `make generate` added this flag. Should I keep it?

Reply:
Yep. Keep it for now.
I'll may adjust the generation logic later to skip this if all flags are hidden.